packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -181,151 +162,103 @@ const Logs: FC<LogsProps> = ({
                                     />
                                 </ActionIcon>
                             </Group>
-                            <Collapse in={openedUuids.has(item.schedulerUuid)}>
+                            <Collapse
+                                in={openedUuids.has(currentLogs[0].jobGroup!)}
```

Comment:
same here if you have `jobGroup` as an argument you can avoid using unsafe operation